cmd/veneur-emit/main.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/Sirupsen/logrus"
+	"github.com/stripe/veneur"
+)
+
+var (
+	configFile = flag.String("f", "", "The Veneur config file to read for settings.")
+	hostport   = flag.String("hostport", "", "Hostname and port of destination. Must be used if config file is not present.")
+	name       = flag.String("name", "", "Name of metric to report. Ex: daemontools.service.starts")
+	gauge      = flag.Float64("gauge", 0, "Report a 'gauge' metric. Value must be float64.")
+	timing     = flag.Duration("timing", time.Now().Sub(time.Now()), "Report a 'timing' metric. Value must be parseable by time.ParseDuration.")
+	timeinms   = flag.Float64("timeinms", 0, "Report a 'timing' metric, in milliseconds. Value must be float64.")
```

Comment:
I think we should leave out the `timeinms` here. Imo — and I'm biased since [I added the other function](https://github.com/DataDog/datadog-go/commit/8b43f0c3bf492206b517f990b01a5ce0943844c1) — the distinction just creates confusion. Original StatsD seems to have some affinity for milliseconds but there's no reason to use me since Datadog is unitless. This leaves the user with one option.